src/linker/Linker/Driver.cs

```diff
@@ -725,6 +721,7 @@ protected int SetupContext (ILogger customLogger = null)
 			// MarkStep
 			// ReflectionBlockedStep [optional]
 			// ProcessWarningsStep
+			// OutputWarningSuppressions
```

Comment:
nit: I think this doesn't match its actual place in the pipeline.

Reply:
yeah, forgot to update the code. The comment is correct.